script.js

```diff
@@ -0,0 +1,93 @@
+// the book object
+const book = {
+  title: '',
+  author: '',
+};
+
+// Check local storage for books collection object
+let booksCollection = JSON.parse(localStorage.getItem('books')) || [];
+
+// Save books to local storage
+function saveBooks(books) {
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+// Remove an existing book function
+function removeBook(title) {
+  booksCollection = booksCollection.filter((obj) => obj.title !== title);
+}
```

Comment:
- [ ] Nice one using JS to add and remove elements dynamically, but please ensure to remove based on DOM and not based on the element's title, this will lead to removing all books with the same title from the list. 
This could also be possible by ensuring you give each added book an ID, that is why ID's are essential, because no two element will have the same ID. 😉 
![mobile](https://user-images.githubusercontent.com/41163571/234016392-7738eee2-7849-4f82-8d3a-88289561e3cd.gif)